vck/src/commonMain/kotlin/at/asitplus/wallet/lib/data/rfc/tokenStatusList/TokenStatusListRfcCwtStatusMechanismProvider.kt

```diff
@@ -0,0 +1,5 @@
+package at.asitplus.wallet.lib.data.rfc.tokenStatusList
+
+import at.asitplus.wallet.lib.data.rfc.tokenStatusList.status.CwtStatusMechanismProvider
+
+interface TokenStatusListRfcCwtStatusMechanismProvider : CwtStatusMechanismProvider
```

Comment:
This interface is actually unused, and if we delete it probably others like `CwtStatusMechanismProvider` and so on, too

Reply:
removed